src/components/transactions/Withdraw/WithdrawModalContent.tsx

```diff
@@ -153,6 +153,7 @@ export const WithdrawModalContent = ({
   if (withdrawTxState.success)
     return (
       <TxSuccessView
+        txHash={withdrawTxState.txHash || ''}
```

Comment:
Since in all the use cases for the `<TxSuccessView />` component it seems that the `txHash` prop is really supporting the same thing in each case, it makes more sense to have an empty string as the fallback at the component level and remove the `|| ''` on all these instantiations of it.  it can be updated with something like:

```js
export type SuccessTxViewProps = {
  txHash?: string;
  // ...
}

export const TxSuccessView = ({
  txHash = '',
  // ...
}: SuccessTxViewProps) => {
  // ...
}

// use
<TxSuccessView
  txHash={withdrawtxState.txHash}
  // ...
/>
```

Reply:
ah yeah good catch. My intention here was to make it required so I could find all of the errors, but I never circled back to clean that up.

But as I was making this change, I realized that all other uses of this modal can stay the same, and the success view default to using the tx state from the provider, if there wasn't a tx hash prop passed in.